src/api/handlers.js

```diff
@@ -0,0 +1,34 @@
+var req_validations = require("./validations.js").request_validations()
+
+module.exports.request_handlers = function() {
+	return {
+		handle_stats_ingestion: function(stat) {
+			return new Promise(function(resolve, reject) {
+				// Validate Params
+				var valid = req_validations.validate_stat(stat)
+				if (valid instanceof Error) {
+					reject(
+						"Post request failed with error. Err:",
+						valid.toString()
+					)
+				} else {
+					if (valid) {
+						let msg = {
+							result: "Success"
+						}
+						resolve(JSON.stringify(msg))
+					} else {
+						reject("Stat " + JSON.stringify(stat) + " is not of supported type")
+					}
+					// ToDo(dev-gaur):
+					// 1. Check if stat is required by an experiment
```

Comment:
May not be relevant now with each of them broken down into separate micro-services.
Have a look and remove those that don't fit in here